content/pages/about.html

```diff
@@ -0,0 +1,396 @@
+<html>
+  <head>
+    <title>About</title>
+    <meta name="template" content="page" />
+    <meta name="status" content="hidden" />
+    <meta name="slug" content="about" />
+  </head>
+  <body>
+
+{% import "partials/homepage.html" as homepage %}
+{% import "partials/about.html" as about %}
+
+<section>
+  <div class="ui horizontally very padded container">
+    <div class="ui very padded centered grid">
+      <div class="middle aligned row">
+
+        <div class="ten wide computer sixteen wide tablet sixteen wide mobile column">
+          <div class="ui large left aligned basic vertical segment">
+            <h1 class="ui huge header">
+              How documentation automation works with Read the Docs
```

Comment:
```suggestion
              Automate your documentation deployment like your code
```

I don't think we need the brand in the h1, we should talk more to the benefits to the user.

Reply:
I was more looking for a phrase like "documentation automation", rather than "automate your documentation", for seo purposes. Agreed the brand name doesn't need to be in there though.

I'm not 100% on the suggestion, but it's in a better direction.

Reply:
For purposes of this header, it needs to also communicate "we're talking about how RTD works". So that also lead me a little away from a generic "Automate your documentation".

I suppose some middle ground options are:

```suggestion
              How documentation automation works
```

```suggestion
              How to automate your documentation deployments
```

```suggestion
              How automatic documentation deployment works
```

Thoughts?

Reply:
I don't feel too strongly. Let's just go with one and get feedback on it. We can always test other language on specific landing pages (eg. Automated Sphinx deployment for your company).

Reply:
Aye, makes sense. I agree dropping the brand sounds best. Maybe start off with the third option for now